docs/fsharp/tutorials/asynchronous-and-concurrent-programming/async.md

```diff
@@ -1,213 +1,349 @@
 ---
-title: Async Programming
-description: Learn how F# async programming is accomplished via a language-level programming model that is easy to use and natural to the language.
-ms.date: 06/20/2016
+title: Async Programming in F#
+description: Learn how F# async programming is accomplished via a language-level programming model that utilizes core functional programming concepts to allow for easy use of asynchrony in F# programming.
+ms.date: 12/17/2018
 ---
 # Async Programming in F# #
 
-> [!NOTE]
-> Some inaccuracies have been discovered in this article.  It is being rewritten.  See [Issue #666](https://github.com/dotnet/docs/issues/666) to learn about the changes.
+Asynchronous programming is essential to modern applications. The applications for asynchronous programming are large in number. There are two primary use cases that most developers will encounter:
 
-Async programming in F# can be accomplished through a language-level programming model designed to be easy to use and natural to the language.
+* Having a non-blocking server process that can service incoming requests while awaiting results for other requests
```

Comment:
Presenting a server process that can service a significant number of parallel incoming requests, while minimizing the system resources locked (reserved/goto thesaurus) while request processing awaits inputs from systems or services external to that process.